NineChronicles.Headless/GraphTypes/StateQuery.cs

```diff
@@ -3,14 +3,13 @@
 using GraphQL.Types;
 using Libplanet;
 using Libplanet.Action;
-using Libplanet.Blockchain;
 using Nekoyume;
 using Nekoyume.Model.State;
 using NineChronicles.Headless.GraphTypes.States;
 
 namespace NineChronicles.Headless.GraphTypes
 {
-    public class StateQuery<T> : ObjectGraphType<BlockChain<T>>
+    public class StateQuery<T> : ObjectGraphType<AccountStateGetter>
```

Comment:
It seems not needed `T` type parameter to me. Are there lines to use it?

Reply:
You're right, thanks for catching that! I'll fix that part.